pwa/src/templates/requestDetailTemplate/RequestDetailTemplate.tsx

```diff
@@ -12,21 +12,25 @@ import {
   UnorderedListItem,
   Link,
 } from "@utrecht/component-library-react/dist/css-module";
-import { TEMP_OBJECTS } from "../../data/detail";
 import { translateDate } from "../../services/dateFormat";
 import { useTranslation } from "react-i18next";
 import { navigate } from "gatsby";
 import { FontAwesomeIcon } from "@fortawesome/react-fontawesome";
 import { faArrowLeft } from "@fortawesome/free-solid-svg-icons";
+import { QueryClient } from "react-query";
+import { useWooRequests } from "../../hooks/wooRequests";
+import Skeleton from "react-loading-skeleton";
+import { getPDFName } from "../../services/getPDFName";
 
-interface ObjectDetailTemplateProps {
-  objectId: string;
+interface RequestDetailTemplateProps {
+  requestId: string;
 }
 
-export const ObjectDetailTemplate: React.FC<ObjectDetailTemplateProps> = ({ objectId }) => {
+export const RequestDetailTemplate: React.FC<RequestDetailTemplateProps> = ({ requestId }) => {

```

Comment:
I'm unsure if _request_ is the right name for this. It's the "Wet open overheid", doesn't have much to do with a _request_, I think.

Maybe something like `WOOItem` would be more descriptive?